src/main/java/us/ihmc/psyonicros2/AbilityHandManager.java

```diff
@@ -214,6 +214,13 @@ private void updateGripControl()
 
             return;
          }
+         else
+         {
+            for (int i = 0; i < ACTUATOR_COUNT; ++i)
+            {
+                  hand.setCommandValue(i, 0.0f);
+            }
+         }
```

Comment:
You can move this out of the else block since it returns in the if. 
```suggestion
         for (int i = 0; i < ACTUATOR_COUNT; ++i)
            hand.setCommandValue(i, 0.0f);
```